docs/dev-guide/contributing/bugs.rst

```diff
@@ -91,6 +95,7 @@ week's new bugs. Bugs that need additional information will have notes put onto
 the issue asking for input. Unless a Redmine user specifically disabled e-mail
 support, adding a note will e-mail the reporter. Bugs with enough information
 will have their severity and priority set, as well as a component if appropriate.
+Also, a target release can be set early on. This way an issue can block a release.
```

Comment:
s/early on/during triage/
s/This way an issue can/An issue that has target release set during triage should/